jmix-ui/ui/src/main/java/io/jmix/ui/widget/client/fileupload/JmixFileUploadState.java

```diff
@@ -61,4 +61,10 @@ public class JmixFileUploadState extends TabIndexState {
 
     @NoLayout
     public String dropZonePrompt;
+
+    @NoLayout
+    public boolean totalProgressDisplayEnabled = true;
```

Comment:
Let's make it `false` by default and only set to `true` for MultiUpload. Single Upload is used more frequently.